### DISCUSSION TITLE
Bluetooth | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#858] [Q&A]
# Bluetooth | Seeed Studio Wiki

WiFi Usage on Seeed Studio XIAO ESP32C3

https://wiki.seeedstudio.com/XIAO_ESP32C3_Bluetooth_Usage/

**Comment by bobwu0214** [comment]
ESP32C3 if it can send data to Arduino uno r4

**Reply by liu2548253579** [reply]
Theoretically it is possible, whether it is wired or wireless, it depends on how you let them create a link and develop a good communication protocol

**Comment by UgunsX** [comment]
I was trying bluetooth server example and it didn't work. Seems like board is not advertising while connected to PC with USB cable. If board is powered from power bank, it is discovered in Light Blue app and can be connected, but com port, of course, is not available in this case. As soon as connected back to PC with usb cable, MyESP32 disappears from device list and can not be connected with phone.

**Reply by liu2548253579** [reply]
Can you tell me which example it is? Maybe I can help you troubleshoot the problem.
